src/modules/LanItems.rb

```diff
@@ -2249,11 +2253,14 @@ def ProposeItem
       @ipaddr = ""
       @netmask = ""
       @bootproto = "dhcp"
+# storage-ng
+=begin
       # #176804
       if Storage.IsDeviceOnNetwork(NetworkStorage.getDevice("/")) != :no
         @startmode = "nfsroot"
         Builtins.y2milestone("startmode nfsroot")
       end
+=end
```

Comment:
Hmmm, this is suspicious. In theory this functionality was already taken back by @gilsonsouza, so I wouldn't expect commented code anymore. Let me check